config.yml

```diff
@@ -131,3 +131,38 @@ pec:
       - "libraryPreparationMethod"
       - "runType"
   contact_email: "kelsey@synapse.org"
+
+annotations-research:
+  parent: "syn21782020"
```

Comment:
I don't have access to this folder. May want to set permissions for the same teams.

Reply:
It is accessible to the Sage Bionetworks team, can you request to join? You should be on there anyway. https://www.synapse.org/#!Team:273957

Reply:
Oh, I thought I was on that team. I just requested access. Thanks!